tests/test_data_grabber.py

```diff
@@ -135,3 +135,25 @@ def test_feature_listing_runtime():
     assert len(tensed_features) > 2
     assert len(interventions) > 2
     assert len(outcomes) > 2
+
+
+def test_no_ma_strings_in_features():
+    features = list_available_features()
+    assert all(not feature.endswith("_ma") for feature in features)
+
+def test_ma_strings_in_features():
+    feature_ma = list_available_features('msa')
+    assert all(feature.endswith("_ma") for feature in feature_ma)
+
+
+feature_ma = list_available_features('msa')
+data = MSADataGrabber()
```

Comment:
Most of the tests performed on `DataGrabber` in `test_DataGrabber()` are now not run on `MSADataGrabber`. Refactor this test by parametrizing it by which data grabber is used and applying all checks that make sense to the latter as well (and the other ones conditionally if dealing with the former).

Reply:
done